lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -0,0 +1,75 @@
+// LMS -- modules -- student notes
+// ====================
+
+// in this document:
+// --------------------
+// +notes
+// +toggling notes
+// +creating/editing notes
+// +listing notes
+
+// +notes:
+// --------------------
+// this Sass partial contains all of the styling needed for the in-line student notes UI.
+
+// +toggling notes
+// --------------------
+
+.edx-notes-visibility {
+
+  .edx-notes-visibility-error {
+    @extend %t-copy-sub2;
+    @extend %text-truncated;
+    position: relative;
+    bottom: -($baseline/20); // needed to sync up with current rogue/more complex calc utility alignment
+    max-width: ($baseline*15);
+    display: none;
+    vertical-align: bottom;
+    @include margin-right(-($baseline/4));
```

Comment:
Yep, in general ``@include X()`` and ``@extend Y`` are listed first and foremost, this is a special case where we're using the RTL mixins that @frrrances put into place for left/right margins, padding, and positioning. In these cases, we leave the rule where it would have been placed without the mixin.